Update data.yaml

iProdigy opened the pull request:
### Description of your *pull request*, and other information
* Update statuses
* Update sources
* Remove redundant promises
* Remove entries that are not measurable promises

---

- [x] I've **verified** and **I assure** that I'm looking at the latest version of Trump Tracker.
- [x] At least skimmed through [README](https://github.com/TrumpTracker/trumptracker.github.io/blob/master/README.md) and **most notably** the [To Do List](https://github.com/TrumpTracker/trumptracker.github.io#to-do-list) and [CONTRIBUTING.md](https://github.com/TrumpTracker/trumptracker.github.io/blob/master/CONTRIBUTING.md) pages.
- [x] [Searched](https://github.com/TrumpTracker/trumptracker.github.io/pulls?utf8=%E2%9C%93&q=) through previous Pull Requests for similar issues including closed ones or just looking at the promises template

iProdigy committed:
update statuses+sources, reduce redundancy

iProdigy committed:
Merge remote-tracking branch 'upstream/master'

iProdigy committed:
update data.yaml

iProdigy committed:
update data.yaml

VirenMohindra reviewed (changes requested):
could you please provide an explanation for removing these policies?

- shut down parts of the internet
- both the minimum wage promises (I understand redundancy, but shouldn't one be kept?)
- allow concealed-carry permits to be recognized in all states
- grow the DFP by at least 6%

thanks

iProdigy commented:
I removed `shut down parts of the internet` because the promise `Shut down parts of the Internet so that Islamic State terrorists cannot use it to recruit American children` already exists. 

I removed the two older minimum wage promises because a promise states `Raise it (the federal minimum wage) to $10 an hour, ignore what I said before (get rid of it)`.

I removed the concealed carry promise because another one already exists (`Allow concealed carry permits to be recognized in all 50 states`).

I removed `grow the DFP by at least 6%` because `DFP` is a typo of `GDP` and the promise is already covered by `Grow the nation''s economy by at least 6 percent.`

VirenMohindra commented:
sounds good!
